Misc/aflytning-keyboard-warrior/index.md

```diff
@@ -0,0 +1,209 @@
++++
+title = 'Aflytning: keyboard warrior'
+categories = ['Misc']
+date = 2023-12-17T20:37:16+01:00
+scrollToTop = true
++++
+
+## Challenge Name:
+
+Aflytning: keyboard warrior
+
+## Category:
+
+Misc
+
+## Challenge Description:
+```
```

Comment:
I would strongly suggest either adding the specific text styling to the section or simply not wrapping it at all, seeing as no specific definition of either code (like python etc.) or text will result in the following layout
![image](https://github.com/onero/CTF-NC3-2023/assets/13084604/29a0869c-1001-44d5-b7cb-9a2c13b5a395)
```suggestion
```text
```

Reply:
Check :) That's not how it looks in my .md viewer in VsCode, which is why I've been ignoring it:
![image](https://github.com/onero/CTF-NC3-2023/assets/483707/455854c5-230b-42bb-a523-01d973fa014e)